lib/components/app/call-taker-panel.js

```diff
@@ -0,0 +1,677 @@
+                  showClearButton={!mobile}
+                />
+              )
+            })}
+            <LocationField
+              inputPlaceholder={`Enter destination or ${actionText} on map...`}
+              locationType='to'
+              showClearButton={!mobile}
+            />
+            <div
+              // FIXME
+              style={{
+                top: '20px'
+              }}
+              className='switch-button-container'>
+              <SwitchButton content={<i className='fa fa-exchange fa-rotate-90' />} />
+            </div>
+            {datastoreUrl
+              ? <>
+                <button
+                  className='clear-button-formatting'
+                  style={{marginBottom: '5px', marginLeft: '10px'}}
+                  onClick={this._addPlace}>
+                  <i className='fa fa-plus-circle' /> Add place
+                </button>
+                <div className='search-options' style={{height: '30px'}}>
+                  <DateTimeOptions
+                    date={date}
+                    onKeyDown={this._handleFormKeyDown}
+                    departArrive={departArrive}
+                    setQueryParam={setQueryParam}
+                    time={time} />
+                  <select
+                    onBlur={this._setMode}
+                    onKeyDown={this._handleFormKeyDown}
+                    value={this.modeToOptionValue(mode)}
+                    style={{position: 'absolute', right: '50px'}}
+                    onChange={this._setMode}>
+                    {modeOptions.map(o => (
+                      <option key={o.value} {...o} />
+                    ))}
+                  </select>
+                  <Button
+                    bsStyle='default'
+                    bsSize='small'
+                    onClick={this._planTrip}
+                    style={{
+                      position: 'absolute',
+                      right: '0px'
+                    }} >
+                    Plan
+                  </Button>
+                </div>
+                <div className='advanced-search-options-container'>
+                  <Button
+                    className='hide-button clear-button-formatting'
+                    onClick={this._onHideAdvancedClick}>
+                    <i className={`fa fa-caret-${expandAdvanced ? 'up' : 'down'}`} /> Advanced options
+                  </Button>
+                  <div className='advanced-search-options' style={!expandAdvanced ? {display: 'none'} : null}>
+                    <CallTakerAdvancedOptions
+                      modes={modes}
+                      ModeIcon={ModeIcon}
+                      routes={routes}
+                      findRoutes={this.props.findRoutes}
+                      setQueryParam={setQueryParam}
+                      currentQuery={currentQuery} />
+                  </div>
+                </div>
+              </>
+              : <>
```

Comment:
Landon to refactor this to be a bit so that FDOT vs. calltaker components are not handled this way.